scss/modules/_forms.scss

```diff
@@ -58,57 +90,6 @@
     }
   }
 
-  input:focus,
-  textarea:focus {
-    border-color: $brand-color;
```

Comment:
@richmccartney cool but if you select a input it just removed the border. I think it needs to change to border color.